src/resources/SearchAnalysis/SearchAnalysis.ts

```diff
@@ -0,0 +1,15 @@
+import dayjs from 'dayjs';
+import API from '../../APICore.js';
+import Resource from '../Resource.js';
+import {ReplayAnalysis} from './SearchAnalysisInterface.js';
+
+export default class SearchAnalysis extends Resource {
+    static baseUrl = '/rest/search/v3/analysis';
+
+    replay(id: string, from = dayjs().subtract(1, 'week').format('YYYY-MM-DD')) {
```

Comment:
I have to pull the break here. I don't think it's a good idea to add dayjs in the library for such little usage.

Dayjs doesn't seems to plan a transition to ESM, which may create issue on NodeJS environment, see https://github.com/iamkun/dayjs/issues/1765

Reply:
Given that the API consume an ISO Date,
```ts
const date = new Date();
date.setDate(date.getDate()-7)
console.log(date.toISOString())
```